src/EditorFeatures/CSharpTest/UsePatternMatching/CSharpAsAndNullCheckTests.cs

```diff
@@ -508,6 +508,55 @@ void M()
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInlineTypeCheck)]
+        [WorkItem(28821, "https://github.com/dotnet/roslyn/issues/28821")]
+        public async Task TestDefiniteAssignment8()
+        {
+            await TestMissingInRegularAndScriptAsync(
+@"class Program
+{
+    static void Foo(System.Activator bar)
```

Comment:
>Foo [](start = 16, length = 3)

We avoid "Foo" in the codebase, as it triggers a vocabulary tool.
I'd suggest `M` or `Method`, but many people change it to `Goo` which is fine too.